Author: Dariusz Dwornikowski <dariusz.dwornikowski@cs.put.poznan.pl>
Subject: Fix building by Debian tools
Forwarded: no
Last-Update: 2015-10-15
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -97,9 +97,9 @@
     set(SOURCES ${SOURCES} recv-pcap.c)
 endif()
 
-configure_file(topt.ggo.in ${CMAKE_BINARY_DIR}/src/topt.ggo @ONLY)
-configure_file(zbopt.ggo.in ${CMAKE_BINARY_DIR}/src/zbopt.ggo @ONLY)
-configure_file(zopt.ggo.in ${CMAKE_BINARY_DIR}/src/zopt.ggo @ONLY)
+configure_file(topt.ggo.in ${CMAKE_SOURCE_DIR}/src/topt.ggo @ONLY)
+configure_file(zbopt.ggo.in ${CMAKE_SOURCE_DIR}/src/zbopt.ggo @ONLY)
+configure_file(zopt.ggo.in ${CMAKE_SOURCE_DIR}/src/zopt.ggo @ONLY)
 
 add_custom_command(OUTPUT zopt.h
     COMMAND gengetopt -C --no-help --no-version --unamed-opts=SUBNETS -i "${CMAKE_CURRENT_SOURCE_DIR}/zopt.ggo" -F "${CMAKE_CURRENT_BINARY_DIR}/zopt"
